cdap-app-templates/cdap-etl/cdap-etl-api/src/main/java/co/cask/cdap/templates/etl/api/batch/BatchSource.java

```diff
@@ -41,4 +46,28 @@ public void configure(StageConfigurer configurer) {
    * @param context {@link BatchSourceContext}
    */
   public abstract void prepareJob(BatchSourceContext context);
+
+  protected long parseFrequency(String frequencyStr) {
```

Comment:
Gokul: This method is copied from ConversionConfig which is in cdap-adapter modules. That module is going to go away right ?